gradle/libs.versions.toml

```diff
@@ -11,6 +11,9 @@ gradleplugin-animal-sniffer = "1.7.2"
 gradleplugin-aQute-bnd = "7.0.0"
 gradleplugin-errorprone = "4.1.0"
 gradleplugin-spotless = "6.25.0"
+gradleplugin-nexusPublish = "2.0.0-rc-1"
+gradleplugin-shipkit-changelog = "2.0.1"
+gradleplugin-shipkit-autoVersion = "2.1.0"
```

Comment:
Nit: since there is only 1 plugin that uses this version, I think it is duplicate to put it here as well. Therefore, if there is only one artifact with a particular version reference, I would prefer to use `version` instead of `version.ref`